### PR TITLE
Omer 1

### PR DESCRIPTION
**Pull request by yasanomer**
This is Omer US2-AS2 assignment. Check pls

### CODE REVIEW

File: src/test/java/com/library/pages/UserPage_Omer.java [hunk 1]
```diff
@@ -30,6 +31,9 @@ public void verifyUserManagementTitle (){
```
Comment (Zebo1985) [inline src/test/java/com/library/pages/UserPage_Omer.java:31]: private WebElement simple

File: src/test/java/com/library/pages/UserPage_Omer.java [hunk 2]
```diff
@@ -30,6 +31,9 @@ public void verifyUserManagementTitle (){
 
         Assert.assertEquals(expected,actual);
         System.out.println("System display successfully User Page ");
+
+
```
Comment (Zebo1985) [inline src/test/java/com/library/pages/UserPage_Omer.java:35]: statik

File: src/test/java/com/library/pages/UserPage_Omer.java (hunk 1 above)
Comment (yasanomer) [reply src/test/java/com/library/pages/UserPage_Omer.java:31]: Ok. This is great idea

File: src/test/java/com/library/pages/UserPage_Omer.java (hunk 2 above)
Comment (Zebo1985) [reply src/test/java/com/library/pages/UserPage_Omer.java:35]: can you check

Comment (yasanomer) [reply src/test/java/com/library/pages/UserPage_Omer.java:35]: last change

**Pull request merged**